src/util.ts

```diff
@@ -58,3 +58,12 @@ export const getUserAgent = (): string => {
   const archName = os.arch();
   return `sigstore-js/${packageVersion} (Node ${nodeVersion}) (${platformName}/${archName})`;
 };
+
+// DSSE Pre-Authentication Encoding
+export function pae(payloadType: string, payload: Buffer): Buffer {
```

Comment:
I tried to find another home for this and understand why you landed it here. I wish there was a better place for it to live. Or maybe a name that's more clear?

Reply:
Yeah, sorta hate the "util" junk drawer, but it's the best I've got for now. The bad name is a good call out though. Updated to something more descriptive.